qemu/tests/hpt_miscellaneous.py

```diff
@@ -88,6 +102,14 @@ def run(test, params, env):
         # For HPT reszing after hotplug memory
         hpt_mem = MemoryHotplugTest(test, params, env)
         hpt_mem.hotplug_memory(vm, "hpt_mem")
+        orig_mem = int(session.cmd_output(cmd=params['free_mem_cmd']))
```

Comment:
Should do this before `hpt_mem.hotplug_memory`, right?

Reply:
So sorry, I didn't notice it then approved it.

Reply:
Already change it before hotplug memory,and thanks.

Reply:
Could you please help to review it ? Thanks. @luckyh